ports/esp32/boards/ESP32_QEMU_C3/board.json

```diff
@@ -0,0 +1,19 @@
+{
+    "deploy": [
+        "../deploy_c3.md"
+    ],
+    "docs": "",
+    "features": [
+        "BLE",
+        "External Flash",
+        "WiFi"
+    ],
+    "images": [
+        "esp32c3_devkitmini.jpg"
+    ],
+    "mcu": "esp32c3",
+    "product": "ESP32-C3",
+    "thumbnail": "",
+    "url": "https://www.espressif.com/en/products/modules",
```

Comment:
Suggest changing some of these to reflect what this actually is,

i.e. `url` can link to https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/api-guides/tools/qemu.html, `product` can be "ESP32-C3 QEMU Emulator", etc.

Reply:
changed to board variant instead